instrumentation/elasticsearch/elasticsearch-api-client-8.0/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("co.elastic.clients")
+    module.set("elasticsearch-java")
+    versions.set("[7.16,)")
```

Comment:
Does the instrumentation work with `7.16.0`?

Reply:
Yes, it work for 7.16+ , 8+.

Do you think we need an additional test module to test with 7.16 explicitly?

Reply:
Our usual pattern is naming the module based on the earliest supported version and testing against it (if multiple versions are needed then `JvmTestSuite` can be used). Lets see what others have to say about this.

Reply:
renamed to 7.16